cython_setuptools/__init__.py

```diff
@@ -1,3 +1,2 @@
-__version__ = "0.2.3"
-
-from .vendor import setup  # noqa
+from ._version import __version__
```

Comment:
Explain in commit message how it works.

Reply:
👍

Reply:
How do we create a version ?

Reply:
A specific version is created with a git tag.
After a target version, all the additional commits are taken in considerations in the `minor` version plus an hash: 

automatic version after an additional commit on tagged version `0.3.0`: 

` 0.3.1.dev1+gbc4cc18`